server/controllers/finance/reports/invoices/creditNote.handlebars

```diff
@@ -145,7 +145,7 @@
 
   <div class="row">
     <div class="col-xs-5">
-      {{translate 'FORM.LABELS.SALE_RESPONSIBLE'}} <br />
```

Comment:
If `SALE_RESPONSIBLE` is no longer used in the application, could you remove it from the `i18n` JSON files?  I did a quick search and couldn't find it used anywhere else in the application.

Reply:
All right